packages/deployments/contracts/contracts_forge/core/connext/libraries/AmplificationUtils.t.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.14;
+
+import "../../../../contracts/core/connext/libraries/SwapUtils.sol";
+import "../facets/FacetHelper.sol";
+
+contract AmplificationUtilsTest is FacetHelper {
+  // ============ Storage ============
+  SwapUtils.Swap swap;
+
+  // ============ Events ============
+  event RampA(uint256 oldA, uint256 newA, uint256 initialTime, uint256 futureTime);
+  event StopRampA(uint256 currentA, uint256 time);
+
+  // ============ Setup ============
+  function setUp() public {
+    IERC20[] memory _pooledTokens = new IERC20[](1);
+    _pooledTokens[0] = IERC20(_adopted);
+    uint256[] memory _tokenPrecisionMultipliers = new uint256[](1);
+    _tokenPrecisionMultipliers[0] = 1;
+    uint256[] memory _balances = new uint256[](1);
+    _balances[0] = 100;
+    swap = SwapUtils.Swap({
+      initialA: 1000,
+      futureA: 10000,
+      initialATime: 100,
+      futureATime: 1000,
+      swapFee: 0,
+      adminFee: 0,
+      lpToken: LPToken(address(100)),
+      pooledTokens: _pooledTokens,
+      tokenPrecisionMultipliers: _tokenPrecisionMultipliers,
+      balances: _balances
+    });
+  }
+
+  // ============ utils ============
+
+  // ============ getA ============
+  function test_AmplificationUtils_getA_works() public {
+    vm.warp(500);
+    assertEq(AmplificationUtils.getA(swap), 50);
+  }
+
+  // ============ getAPrecise ============
+  function test_AmplificationUtils_getAPrecise_works() public {
+    vm.warp(500);
+    assertEq(AmplificationUtils.getAPrecise(swap), 5000);
+  }
```

Comment:
Same ^^

Reply:
updated in the commit a2e74c88e3600abb7f2b57cf4f53fdc67f6f823e